app/actions/views/file_upload.js

```diff
@@ -1,10 +1,8 @@
 // Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
-import FormData from 'form-data';
```

Comment:
Can we actually remove the `form-data` library as a dependency in package.json? I don't think we'd use it anywhere else.

Reply:
Weird thing is it was only a devDependency...?